src/components/Form/FormLabel.tsx

```diff
@@ -1,29 +1,33 @@
-import { Inject } from 'vue-property-decorator';
-import { Component, DefaultSlot, Prop, Base } from '@/core';
-import { FORM_ITEM_ID_KEY } from './FormItem/Types';
+import {Inject} from 'vue-property-decorator';
+import {Component, DefaultSlot, Prop, Base} from '@/core';
+import {FORM_ITEM_ID_KEY} from './FormItem/Types';
 
 interface Props {
   for?: string | null;
   required?: boolean;
 }
 
 @Component('FormLabel')
-@DefaultSlot('Contents of the label: For non-inline elements simply use text which will become the text displayed by the label. For inline elements use text alongside with any elements that form your input control.')
-export class FormLabel extends Base<Props> {
-  @Inject(FORM_ITEM_ID_KEY) public itemId!: string;
+@DefaultSlot('Contents of the label: For non-inline elements simply use text which will become' +
```

Comment:
I prefer not to use `+` - event for long strings. The reason for that is described in the Airbnb-style guide.

https://github.com/airbnb/javascript#strings--line-length

I found no rule for tslint to enforce that. But I will add this issue to our agenda for the next *meeting*. I don't like the way we currently use tslint and maybe we can - all together - come up with something better.